helpers/lrug_helpers.rb

```diff
@@ -295,6 +295,8 @@ def talks_for_rubyevents_video_playlist(talks, title, meeting_date, published_at
     talks.map do |id, talk|
       video_coverage = talk.coverage&.detect { it.type == 'video' }
       slides_coverage = talk.coverage&.detect { it.type == 'slides' }
+      writeup_coverage = talk.coverage&.detect { it.type == 'write-up' }
+      writeup_description = writeup_coverage ? "\n\nWrite-up: [#{writeup_coverage.title}](#{writeup_coverage.url})" : ""
```

Comment:
Oh, this is neat, I don't think I would have thought of this.  👍

Reply:
Yes, thanks for keeping such a good track record of all the meetup data!

There's also an open issue in RubyEvents that I'm kind of motivated to work on now, so we can properly add this metadata: https://github.com/rubyevents/rubyevents/issues/447